test/ExecutionFramework.cpp

```diff
@@ -210,11 +218,12 @@ void ExecutionFramework::sendEther(h160 const& _addr, u256 const& _amount)
 		if (_amount > 0)
 			cout << " value: " << _amount << endl;
 	}
-	evmc_message message = {};
+	evmc_message message{};
 	message.sender = EVMHost::convertToEVMC(m_sender);
 	message.value = EVMHost::convertToEVMC(_amount);
 	message.kind = EVMC_CALL;
-	message.destination = EVMHost::convertToEVMC(_addr);
+	message.recipient = EVMHost::convertToEVMC(_addr);
+	message.code_address = message.recipient;
```

Comment:
`code_address` is the same as this is a regular call. It only differs for delegatecall/callcode.